src/control/server/testdata/expect_daos_server_psm2.yml

```diff
@@ -39,8 +39,11 @@ hyperthreads: false
 nr_hugepages: 4096
 control_log_mask: DEBUG
 control_log_file: /tmp/daos_control.log
+targets: 0
 modules: ""
 attach: ""
+xshelpernr: 0
```

Comment:
these will be ignored due to what was described in the previous comment (no yaml tag)